File: 898-transpose-matrix/transpose-matrix.java
class Solution {
    public int[][] transpose(int[][] matrix) {
        
        int[][] result = new int[matrix[0].length][matrix.length];
        for(int i =0; i<matrix.length; i++){
            for (int x =0; x<matrix[0].length; x++){
                result [x][i] = matrix[i][x];
            }
        }
        return result;
    }
}

File: 898-transpose-matrix/transpose-matrix.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        
        temp = [[0 for _ in range(len(matrix))]for _ in range(len(matrix[0]))]
        for i in range(len(matrix)):
            #print(matrix[i])
            for x in range(len(matrix[0])):
                temp[x][i]=matrix[i][x]
        return temp